packit/config/job_config.py

```diff
@@ -44,24 +45,21 @@ class JobType(Enum):
     copr_build_started = "copr_build_started"
 
 
-class JobTriggerType(Enum):
+class JobConfigTriggerType(Enum):
     release = "release"
     pull_request = "pull_request"
     commit = "commit"
-    installation = "installation"
-    testing_farm_results = "testing_farm_results"
-    comment = "comment"
```

Comment:
but comment can be a trigger type as well, right?

Reply:
We used it only internally, or not?

I would go with `issue`/`pull-request` here and use commits for retriggering without a need to have it in the config.

Reply:
ah, okay, got it, makes sense